api/src/test/java/org/pmiops/workbench/tools/cdrconfig/CdrConfigVOMapperTest.java

```diff
@@ -68,7 +69,8 @@ public void test_toDbTier() {
             .setDisplayName(testTierJson.displayName)
             .setServicePerimeter(testTierJson.servicePerimeter)
             .setAuthDomainName(testTierJson.authDomainName)
-            .setAuthDomainGroupEmail(testTierJson.authDomainGroupEmail);
+            .setAuthDomainGroupEmail(testTierJson.authDomainGroupEmail)
+            .setEnableUserWorkflows(false);
```

Comment:
this is testing that `enableWorkflows` is always set to false, instead of what it is reading from the JSON in `testTierJson`